docs/proposals/multi-node/README.md

```diff
@@ -1242,7 +1242,7 @@ The life-cycle of these work-flows is shown below.
 
 ### Work-flows independent of leader election in all members
 
-- Serve the HTTP API that all members are expected to support currently
+- Serve the [HTTP API](https://github.com/gardener/etcd-backup-restore/blob/master/pkg/server/httpAPI.go#L101-L107) that all members are expected to support currently but HTTP API call which are used to take [out-of-sync delta or full snapshot](https://github.com/gardener/etcd-backup-restore/blob/master/pkg/server/httpAPI.go#L103-L105) should delegate the incoming HTTP requests to the `leading-sidecar`.
```

Comment:
```suggestion
- Serve the [HTTP API](https://github.com/gardener/etcd-backup-restore/blob/master/pkg/server/httpAPI.go#L101-L107) that all members are expected to support currently but HTTP API call which are used to take [out-of-sync delta or full snapshot](https://github.com/gardener/etcd-backup-restore/blob/5dfcc1f848a9f325d41a24eae4defb70d997c215/pkg/server/httpAPI.go#L103-L105) should delegate the incoming HTTP requests to the `leading-sidecar` via an HTTP reverse proxy.
```

Reply:
> via an HTTP reverse proxy

This is an implementation detail and I don’t think we should mention this in doc. The design document shouldn’t recommend the implementation method unless it is the only method available to achieve the functionality. WDYT?

Reply:
Fair enough, but can you add it as an example -> (e.g. via an HTTP reverse proxy)? The problem today is that we don't have a dedicated documentation about multi-node etcd, only the proposal as a reference document. I think it's beneficial for every interested party to know more details w/o looking at the code.

Reply:
make sense,I will mention this as one of the possible approach.

Reply:
done